app/views/draws/show.html.erb

```diff
@@ -1,3 +1,19 @@
 <h1 class="draw-name"><%= @draw.name %></h1>
+<div class="intent-metrics">
```

Comment:
I guess it's not a problem, I'm just not sure if it's necessary / wanted, but I think we can leave it for now.